diff --git a/server/src/main.ts b/server/src/main.ts
index 4558589..4a0994e 100644
--- a/server/src/main.ts
+++ b/server/src/main.ts
@@ -16,17 +16,17 @@ httpServer.use(express.urlencoded({ extended: true }));
 httpServer
     .listen(
         {
-            host: "::",
+            host: "127.0.0.1",
             port: PORT_NUMBER,
         },
         (err) => {
             if (err) {
                 console.error(
-                    `Could not listen on [::]:${PORT_NUMBER}, falling back to 0.0.0.0 (Caused by ${strerror(err)})`,
+                    `Could not listen on 127.0.0.1:${PORT_NUMBER} (Caused by ${strerror(err)})`,
                 );
             } else {
                 console.log(
-                    `Started POT server (v${VERSION}) on on address [::]:${PORT_NUMBER}`,
+                    `Started POT server (v${VERSION}) on address 127.0.0.1:${PORT_NUMBER}`,
                 );
             }
         },
@@ -36,17 +36,17 @@ httpServer
         // this is temporary as we plan to bind to localhost in the next major version
         httpServer.listen(
             {
-                host: "0.0.0.0",
+                host: "127.0.0.1",
                 port: PORT_NUMBER,
             },
             (err) => {
                 if (err) {
                     console.error(
-                        `Could not listen on [::]:${PORT_NUMBER} (Caused by ${strerror(err)})`,
+                        `Could not listen on 127.0.0.1:${PORT_NUMBER} (Caused by ${strerror(err)})`,
                     );
                 } else {
                     console.log(
-                        `Started POT server (v${VERSION}) on address 0.0.0.0:${PORT_NUMBER}`,
+                        `Started POT server (v${VERSION}) on address 127.0.0.1:${PORT_NUMBER}`,
                     );
                 }
             },
